apps/explorer/src/components/validator-map/index.tsx

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useQuery } from '@tanstack/react-query';
+import { ParentSize } from '@visx/responsive';
+import { TooltipWithBounds, useTooltip } from '@visx/tooltip';
+import React, { type ReactNode, useCallback, useMemo } from 'react';
+
+import { WorldMap } from './WorldMap';
+import { type ValidatorMapData } from './types';
+
+// import { useNetwork } from '~/context';
+import { Card } from '~/ui/Card';
+import { Heading } from '~/ui/Heading';
+import { Placeholder } from '~/ui/Placeholder';
+import { Text } from '~/ui/Text';
+
+// const HOST = 'https://apps-backend.sui.io';
+
+type ValidatorsMap = Record<string, ValidatorMapData>;
+
+const numberFormatter = new Intl.NumberFormat('en');
+
+function NodeStat({ title, children }: { title: string; children: ReactNode }) {
+    return (
+        <div className="space-y-1.5">
+            <Heading variant="heading2/semibold" color="steel-darker">
+                {children}
+            </Heading>
+            <Text variant="caption/semibold" color="steel-dark">
+                {title}
+            </Text>
+        </div>
+    );
+}
+
+interface Props {
+    minHeight: string | number;
+}
+
+// NOTE: This component is lazy imported, so it needs to be default exported:
+export default function ValidatorMap({ minHeight }: Props) {
+    // TODO BEFORE MERGING: update query URI
+    const { data, isLoading, isSuccess } = useQuery(
+        ['validator-map'],
+        async () => {
+            const res = await fetch(
+                `http://localhost:3003/validator-map?network=testnet`,
```

Comment:
ignore - just waiting to verify the deployed apps-be piece to finalize this

Reply:
`useAppsBackend` should give you a fetch function you can use

Reply:
Did `useAppsBackend` not work out here or are we waiting for the PR to land there before updating?